var/spack/repos/builtin/packages/cray-libsci/package.py

```diff
@@ -25,6 +25,8 @@ class CrayLibsci(Package):
     version("16.06.1")
     version("16.03.1")
 
+    conflicts("platform=windows")
```

Comment:
I think we should also conflict on darwin. Newer cray is detected as linux, so we can't conflict on "not cray".